src/data/builder-tools.js

```diff
@@ -972,14 +981,38 @@ export const Showcases = [
     getstarted: null,
     tags: ["http", "typescript", "javascript", "wallet", "serialization", "operatortool"],
   },
-   {
+  {
     title: "DCOne Crypto Webhook API",
     description:"API for developers to receive information on changing stake balance.",
     preview: require("./builder-tools/dconecrypto-webhook.png"),
     website: "https://github.com/DCOneCrypto/StakeAddress-Tracking-Webhook-API",
     getstarted: null,
     tags: ["http"],
-  }
+  },
+  {
+    title: "bech32",
```

Comment:
Since this title appears as a builder tool name and there's already a _bech32_ (`input-output-hk/bech32`), can you call it something distinct like:
- `bech32-crystal`
- `wout/bech32
?

Reply:
Sure, I renamed it to `bech32.cr` since that's often used in the community.